CHANGES

```diff
@@ -9,7 +9,10 @@
 
  Changes between 1.0.2p and 1.0.2q [xx XXX xxxx]
 
-  *)
+  *) Resolve a compatibility issue in EC_GROUP handling with the FIPS Object
+     Module, accidentally introduced while reverting blinding in ECDSA sign and
+     hindering the use of ECC in FIPS mode.
+     [Nicola Tuveri]
```

Comment:
Given that, due to this issue, 1.0.2p appears to have serious compatibility problems when using ECC in FIPS mode, I think a `CHANGES` entry is required, to have a documentation trail for users about the potential for incompatibility in 1.0.2p.

I'm not sure about the wording and the style though, so I'd warmly accept feedback (or complete rewrites) from the reviewers (and in general from native English speakers).

Also, is talking about `EC_GROUP` too technical and in-depth for a `CHANGES` entry?